project/run.py

```diff
@@ -3,16 +3,25 @@
 import pytorch_lightning as pl
 from model import JeopardyModel
 from pytorch_lightning.loggers import WandbLogger
+import subprocess
 
-wandb_logger = WandbLogger(name='Adam-32-0.001',project='contra-pytorchlightning')
+PROJECT_NAME = "contra_pytorchlightning"
+wandb_logger = WandbLogger(name='from checkpoint with different hparams',project=PROJECT_NAME)
```

Comment:
and PROJECT_NAME should have been in here with name=PROJECT_NAME

Reply:
I'd make a third commit but I already rebased and messed around once and 3 in one PR might be too messy